src/i18n/en-US.json

```diff
@@ -727,9 +727,14 @@
   "messageFailedToSendWillNotReceiveSingular": "won't get your message.",
   "messageFailedToSendWillReceivePlural": "will get your message later.",
   "messageFailedToSendWillReceiveSingular": "will get your message later.",
-  "mlsConversationRecovered": "You haven’t used this device for a while, or an issue has occurred. Some older messages may not appear here.",
+  "messageReactionDetails": "{{emojiCount}} reaction, react with {{emojiName}} emoji",
```

Comment:
hummm, this seems unrelated to the topic 🤔

Reply:
No idea what it's doing here, good catch, will remove it.